src/encoder.js

```diff
@@ -112,33 +111,33 @@ class Encoder {
   }
 
   /*
-   * Converts a uint8 to hex padded-left to 32 bytes.
-   * @param uint8 The number to convert.
-   * @return The converted uint8 to padded-left hex string.
+   * Converts a uint to hex padded-left to 32 bytes.
+   * @param num The number to convert.
+   * @return The converted uint to padded-left hex string.
    */
-  static uint8ToHex(uint8) {
-    if (uint8 === undefined) {
-      throw new Error(`uint8 should not be undefined`);
+  static uintToHex(num) {
+    if (!Validator.isNumber(num)) {
+      throw new Error(`num is not a Number`);
     }
 
-    let hexNumber = Web3Utils.toHex(uint8);
+    let hexNumber = Web3Utils.toHex(num);
     return Web3Utils.padLeft(hexNumber, PADDED_BYTES).slice(2);
```

Comment:
@dwalintukan this looks strange. Why `padLeft` and cut left (I mean `slice(2)`), can we handle `slice(2)` inside `padLeft`?

Reply:
Web3Utils is not my library. And padLeft needs a hex number so if I slice before that it won't do the padding. (I am slicing the 0x hex prefix).